samples/suit/flash_companion/README.rst

```diff
@@ -0,0 +1,115 @@
+
+The application is meant to be booted by the secure domain while performing the firmware update process using the :ref:`SUIT <ug_nrf54h20_suit_dfu>` firmware upgrade.
+
+The flash companion firmware is not a stand-alone application, it is intended to be used with the ``nrf54h_suit_sample`` to complete a firmware transfer with external flash.
+The application must be built as a child image of a different sample as it inherits the devicetree configuration from the parent build.
+
+Configuration
+*************
+
+|config|
+
+Setup
+=====
+
+You can build the application using Sysbuild by enabling the :kconfig:option:`SB_CONFIG_SUIT_BUILD_FLASH_COMPANION` Kconfig option.
+The memory partition from which the firmware will run can be configured by providing a device tree overlay through Sysbuild.
+You should create a dedicated partition in non-volatile memory and override the ``zephyr,code-partition``.
+The memory partition must not be used by anything else.
+
+The application is booted during the SUIT firmware upgrade process.
+See the :ref:`ug_nrf54h20_suit_external_memory` user guide to learn how to setup your application to boot and use the flash companion's services during firmware upgrade.
+
+Configuration options
+=====================
+
+Check and configure the following configuration option:
+
+.. _SB_CONFIG_SUIT_BUILD_FLASH_COMPANION:
+
+SB_CONFIG_SUIT_BUILD_FLASH_COMPANION - Configuration for the firmware
+   This option enables the firmware and builds it during the Sysbuild.
+
+Building and running
+********************
+
+The flash companion firmware is not a stand-alone application. To build it, open a different sample or application, such as :file:`samples/suit/smp_transfer`. Make sure that both the main application and the flash companion support your target platform.
+
+Perform the following steps in the main application directory:
+
+1. Enable the :kconfig:option:`SB_CONFIG_SUIT_BUILD_FLASH_COMPANION` Sysbuild option
+#. Create :file:`sysbuild/flash_companion.overlay` device tree overlay file and add the following content:
+   a. Create a dedicated partition in non-volatile memory:
+
+      .. code-block:: devicetree
+
+         &cpuapp_rx_partitions {
+            /delete-node/ partition@a6000;
+
+            cpuapp_slot0_partition: partition@a6000 {
+               reg = <0xa6000 DT_SIZE_K(448)>;
+            };
+
+            companion_partition: partition@116000 {
+               reg = <0x116000 DT_SIZE_K(64)>;
+            };
+         };
+
+      In the above example the executable memory partition of the main application (``cpuapp_slot0_partition``) is shrunk to make space for the flash companion executable memory partition (``companion_partition``).
+
+   #. Apply the same memory partition configuration to the main application device tree overlay.
+
+   #. Enable SPI NOR device tree node. In case of nRF54H20dk you can enable the following node:
```

Comment:
```suggestion
   #. Enable SPI NOR devicetree node.
      In the case of nRF54H20 DK, you can enable the following node:
```

Reply:
This seems to be missed: 

"In the case of **nRF54H20 DK**, you can enable the following node:"